### ISSUE TITLE
Let's Encrypt does not accept *@example.org accounts

### ISSUE THREAD

**Issue by Brugman** [litespeedtech/cyberpanel#1010]
I've been fighting with SSL certs today. Turns out I was getting the following error:

```sh
Register account Error: {
  "type": "urn:ietf:params:acme:error:invalidEmail",
  "detail": "Error creating new account :: invalid contact domain. Contact emails @example.org are forbidden",
  "status": 400
}
```

Because my admin email was set to example@example.org. However, when I changed it to my real email, and re-ran the [manual SSL request](https://community.cyberpanel.net/docs?search=ssl&topic=90#toc-h2-debugging-with-command-line), it still would not work.

I fixed this by running this once:
```sh
/root/.acme.sh/acme.sh --register-account -m REALEMAIL
```

Acme seems to cache your LE "account" so that solved the problem for me. However it got me thinking, if the default email address is now example@example.org, there's going to be a lot of people running into this problem if that email is used for Let's Encrypt.

I found this in the source code, and think it probably needs to be reviewed by an expert.

Setting a valid admin email may have to become a requirement and part of the installation.

```sh
            command = '%s --register-account -m %s' % (acmePath, adminEmail)
```
```sh
    def installSSLForDomain(virtualHostName, adminEmail='example@example.org'):
```

**Comment by usmannasir** [comment]
I just deployed a fresh server and didn't see any problem like this
```

hey.weeio2mlx7.cyberpanel.website has SSL from Let's Encrypt.

Your SSL will expire in 89 days.
```

**Comment by Brugman** [comment]
Can you find and use a command to see what email it registered with?

**Comment by usmannasir** [comment]
Which command?

**Comment by usmannasir** [comment]
which CA are you using?

**Comment by Brugman** [comment]
I was hoping that just like the register command, there would be a command to show who is currently registered, and what email that is.

I'm trying some stuff from the [opions](https://github.com/acmesh-official/acme.sh/wiki/Options-and-Params) docs but I'm not getting that info.

```
$ /root/.acme.sh/acme.sh --update-account
[Tue 27 Dec 2022 07:56:26 PM CET] Error. The account was not updated.

$ /root/.acme.sh/acme.sh --register-account
[Tue 27 Dec 2022 07:56:47 PM CET] Registering account: https://acme-v02.api.letsencrypt.org/directory
[Tue 27 Dec 2022 07:56:48 PM CET] Already registered
[Tue 27 Dec 2022 07:56:48 PM CET] ACCOUNT_THUMBPRINT='11111111111111111REDACTED111111111111111111'

$ /root/.acme.sh/acme.sh --register-account -m
[Tue 27 Dec 2022 07:58:57 PM CET] Registering account: https://acme-v02.api.letsencrypt.org/directory
[Tue 27 Dec 2022 07:58:58 PM CET] Already registered
[Tue 27 Dec 2022 07:56:48 PM CET] ACCOUNT_THUMBPRINT='11111111111111111REDACTED111111111111111111'
```

**Comment by usmannasir** [comment]
I don't see an account in account.conf either, but SSL works fine, I remember adding register account for zerossl CA. But we are using LetsEncrypt.

**Comment by Brugman** [comment]
Here's the full error: https://gist.github.com/Brugman/23cb43a10e36b22097186bbb949109d0 (deleting in a week or so)

No mention of ZeroSSL, and I don't recall manually trying to use them after reading about their business model.

**Comment by jcn50** [comment]
The error does not come from the `example@example.org` but from somewhere else, see issue #1013 .

**Comment by Brugman** [comment]
Are you sure? I think that folder is purposely being used for all SSL certs because I did have to add a reference to it in some of my LS vhosts (it's in the docs). So devs are aware of it.

**Comment by jcn50** [comment]
Sorry I hadn't seen those lines in your logs:
```
[Fri 23 Dec 2022 01:25:27 AM CET] Register account Error: {
  "type": "urn:ietf:params:acme:error:invalidEmail",
  "detail": "Error creating new account :: invalid contact domain. Contact emails @example.org are forbidden",
  "status": 400
}
```
So I guess the current issue is also correct!

**Comment by usmannasir** [comment]
@jcn50 you had the same error?

**Comment by jcn50** [comment]
> @jcn50 you had the same error?

@usmannasir no, but the logs of @Brugman was showing the same (wrong) webroot `/usr/local/lsws/Example/html` as my issue #1013 so I thought they could be related or combined~

**Comment by atsigr** [comment]
@usmannasir i had updated the server to latest commit and after a few days the SSL didnt run correctly. I have now manually run to renew SSL on cmd and i get it did success with bellow info , but actually did not work. I mean when i access the site is comes as selvsigned SSL. 

[Mon 13 Feb 2023 01:25:33 PM CET] Cert success.

[Mon 13 Feb 2023 01:25:33 PM CET] Your cert is in: /root/.acme.sh/dataexpert.no_ecc/dataexpert.no.cer
[Mon 13 Feb 2023 01:25:33 PM CET] Your cert key is in: /root/.acme.sh/dataexpert.no_ecc/dataexpert.no.key
[Mon 13 Feb 2023 01:25:33 PM CET] The intermediate CA cert is in: /root/.acme.sh/dataexpert.no_ecc/ca.cer
[Mon 13 Feb 2023 01:25:33 PM CET] And the full chain certs is there: /root/.acme.sh/dataexpert.no_ecc/fullchain.cer
[Mon 13 Feb 2023 01:25:33 PM CET] Installing cert to: /etc/letsencrypt/live/dataexpert.no/cert.pem
[Mon 13 Feb 2023 01:25:33 PM CET] Installing key to: /etc/letsencrypt/live/dataexpert.no/privkey.pem
[Mon 13 Feb 2023 01:25:33 PM CET] Installing full chain to: /etc/letsencrypt/live/dataexpert.no/fullchain.pem
[Mon 13 Feb 2023 01:25:33 PM CET] _on_issue_success

Something is strange because if i check after this success on the website the ssl certificate is old date. 
Something is not pointing to correct SSL maybe ?

**Comment by smyja** [comment]
> @usmannasir i had updated the server to latest commit and after a few days the SSL didnt run correctly. I have now manually run to renew SSL on cmd and i get it did success with bellow info , but actually did not work. I mean when i access the site is comes as selvsigned SSL.
> 
> [Mon 13 Feb 2023 01:25:33 PM CET] Cert success.
> 
> [Mon 13 Feb 2023 01:25:33 PM CET] Your cert is in: /root/.acme.sh/dataexpert.no_ecc/dataexpert.no.cer [Mon 13 Feb 2023 01:25:33 PM CET] Your cert key is in: /root/.acme.sh/dataexpert.no_ecc/dataexpert.no.key [Mon 13 Feb 2023 01:25:33 PM CET] The intermediate CA cert is in: /root/.acme.sh/dataexpert.no_ecc/ca.cer [Mon 13 Feb 2023 01:25:33 PM CET] And the full chain certs is there: /root/.acme.sh/dataexpert.no_ecc/fullchain.cer [Mon 13 Feb 2023 01:25:33 PM CET] Installing cert to: /etc/letsencrypt/live/dataexpert.no/cert.pem [Mon 13 Feb 2023 01:25:33 PM CET] Installing key to: /etc/letsencrypt/live/dataexpert.no/privkey.pem [Mon 13 Feb 2023 01:25:33 PM CET] Installing full chain to: /etc/letsencrypt/live/dataexpert.no/fullchain.pem [Mon 13 Feb 2023 01:25:33 PM CET] _on_issue_success
> 
> Something is strange because if i check after this success on the website the ssl certificate is old date. Something is not pointing to correct SSL maybe ?

I am facing a similar ssl issue for emails. I have updated mulltiple times but emails aren't being sent from gmail. They work with rainloop though.

**Comment by Brugman** [comment]
Do not re-use GitHub issues if your bug is not exactly the same as the OP's. Make a new issue.

**Comment by master3395** [comment]
Thanks for the report. This issue is older and may not apply to current releases.

Please confirm:
1) CyberPanel version + build (please retest on the latest public release **2.4.9**)
2) OS name and version (whatever you run, e.g. `cat /etc/os-release`)
3) Exact steps still failing on a current install
4) Relevant logs (no passwords)

Note: the latest public release is **2.4.9**. Development branches are not recommended for reproducing bugs.

If we do not hear back within 30 days, this issue may be closed as stale. You can reopen with fresh details anytime.
